server/src/models/Expense.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+
+# Location is stored as an x and y coordinate (easier to input into maps APIs)
+# All fields are currently required
+class Expense(BaseModel):
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    # .... the rest of the fields
```

Comment:
Remove this comment. Also, don't directly apply the suggested changes, always check locally to make sure it works. The suggested changes are just to guide you.